Sprint 10 -JS-Functions/tema1.js

```diff
@@ -0,0 +1,9 @@
+// Scrieți o funcție care returnează pătratul unui număr
+// Funcția findSquare(num) returnează pătratul numărului primit ca date de intrare (este transmis ca 
+// argument în apelul funcției).
+
+getSquare = (num) => console.log(num*num);
```

Comment:
functia trebuie sa returneze un numar, nu sa afiseze in consola.

Afisarea se face mai jos